publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -238,14 +239,27 @@ private Publication unpublishPublication(UnpublishPublicationRequest unpublishPu
                                                             existingPublication,
                                                             userInstance);
         resourceService.unpublishPublication(updatedPublication);
-
+        setPendingTicketsToNotRelevant(updatedPublication);
```

Comment:
Maybe we can make resourceService to do it for us? But it is up to you. 

`resourceService.unpublishAllTicketsForPublication(updatedPublication);`

Since we are already fetching this tickets by resourceService, we can do like this:

```
dao.fetchAllTickets(getClient()).stream()
             .map(TicketDao::getData)
             .map(TicketEntry.class::cast)
             .map(ResourceService::setAsNotRelevant)
             .map(Entity::toDao)
             .map(TicketDao.class::cast)
             .forEach(ticketDao -> ticketDao.updateExistingEntry(getClient()));
```